incubator/redis-cache/values.yaml

```diff
@@ -0,0 +1,49 @@
+# Default values for redis-cache.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+##Statefulset related configurations ss.yaml
+replicaCount: 3
+redis:
+  image:
+    repository: redis
+    tag: 3.0-alpine
+    pullPolicy: IfNotPresent
+  resources:
+    limits:
+      memory: 128Mi
+    requests:
+      memory: 128Mi
+microSentinel:
+  image:
+    repository: dhilipkumars/redis-sentinel-k8s
+    tag: latest
```

Comment:
By default it will use the `Always` imagePullPolicy if the tag is `latest`, but you're explicitly setting the pull policy to `IfNotPresent`. So you would need to update the pull policy to `Always` to get that behaviour.